client/app/services/query-result.js

```diff
@@ -224,6 +224,7 @@ function QueryResultService($resource, $timeout, $q) {
         let seriesName;
         let xValue = 0;
         const yValues = {};
+        let eValue = 0;
```

Comment:
I just realized that we will always have an eValue... did you check how it affects queries that don't have a error column set?